src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala

```diff
@@ -322,6 +322,18 @@ class RoutesPostgresRepoSpec extends FunSpec with BeforeAndAfter with Matchers w
         val result: Seq[(RouteRow, PathRow)] = routesRepo.selectFiltered()
         result.size should be (2)
       }
+
+      it("should paginate the result if pagination is provided") {
```

Comment:
it's cool to test with '1', but maybe in this case it would be worth testing with 'some', as well